src/overlays/effects/ovl_Effect_Ss_Extra/z_eff_ss_extra.c

```diff
@@ -1,29 +1,94 @@
 /*
  * File: z_eff_ss_extra.c
  * Overlay: ovl_Effect_Ss_Extra
- * Description:
+ * Description: The floating points hovering over a target you hit in OOT
+ *   IE: 30/60/100 points when hitting the target in lost woods with slingshot to get bigger seed sack
  */
 
 #include "z_eff_ss_extra.h"
+#include "objects/object_yabusame_point/object_yabusame_point.h"
 
 #define PARAMS ((EffectSsExtraInitParams*)initParamsx)
 
-s32 EffectSsExtra_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
+u32 EffectSsExtra_Init(GlobalContext* globalCtx, u32 index, EffectSs* this, void* initParamsx);
 void EffectSsExtra_Update(GlobalContext* globalCtx, u32 index, EffectSs* this);
 void EffectSsExtra_Draw(GlobalContext* globalCtx, u32 index, EffectSs* this);
 
-#if 0
+static s16 gScores[] = { 30, 60, 100 };
```

Comment:
```suggestion
static s16 sScores[] = { 30, 60, 100 };
```
It's just local to this file right?